cmd/thanos/rule.go

```diff
@@ -70,7 +70,7 @@ func registerRule(m map[string]setupFunc, app *kingpin.Application, name string)
 		Default("2h"))
 	tsdbRetention := modelDuration(cmd.Flag("tsdb.retention", "Block retention time on local disk.").
 		Default("48h"))
-
+	//TODO(ivan): add alertmanagers urls to file sd
```

Comment:
Do we want this?

Reply:
Not, at the moment I would say ... lets leave it as it is slightly different to what we are currently changing.